cmd/macoffice/generate.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/fleetdm/fleet/v4/server/vulnerabilities/macoffice"
+)
+
+func panicif(err error) {
+	if err != nil {
+		panic(err)
+	}
+}
+
+// Generates Mac Office release notes metadata in JSON format, to be used by our vulnerability process.
+func main() {
+	wd, err := os.Getwd()
+	panicif(err)
```

Comment:
Question: What do we do if this executable panics?

Reply:
Should we not log the error and exit in a clean way?

Reply:
That's a good question - this executable is meant to be run mainly by the Github workflow and is only one of many 'steps' that we run for generating the metadata that we use for vulnerability processing https://github.com/fleetdm/nvd/blob/master/.github/workflows/generate.yml#L40 - in case of error we want to panic so that the whole 'workflow' fails.